lesson_09/oop/oop_app/src/main/java/com/codedifferently/lesson9/mohamedibrahim/Library.java

```diff
@@ -0,0 +1,117 @@
+package com.codedifferently.lesson9.mohamedibrahim;
+
+import java.util.ArrayList;
+
+public class Library {
+  private ArrayList<Book> shelves = new ArrayList<Book>();
```

Comment:
Should just be `books` since that's what it is. I'd expect shelves to return a `Map` of book lists keyed by shelf ID, tag, or category name.